pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/DataBlockCache.java

```diff
@@ -30,6 +30,7 @@
  * data multiple times. This class allocate resources on demand, and reuse them as much as possible to prevent garbage
  * collection.
  */
+@SuppressWarnings("Duplicates")
```

Comment:
Well, somehow IDE thinks that the code are duplicate, even though the data type of the two are different (one is [], another is [][]). Cleaning up MV will eliminate [][].